aws/deep-learning-containers
# Rely only on github actions for formatting

*GitHub Issue #, if available:*

**Note**: 
- If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

- All PR's are checked weekly for staleness. This PR will be closed if not updated in 30 days.

### Description
- Remove quick checks test
- Add formatting commit to ignore
- change name slightly of github action
- after merge, will make this github action a required job

### Tests run
- Run github action tests
- Ensure failure works ([0a4a600](https://github.com/aws/deep-learning-containers/pull/2918/commits/0a4a6000712ec1f9e1f9de67571d12078aa383fe))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
